build_projects/shared-build-targets-utils/Utils/BuildVersion.cs

```diff
@@ -7,7 +7,7 @@ public class BuildVersion : Version
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
```

Comment:
What uses it?  I had the impression this was used to represent the nuget version picked up which isn't yet stable, but that assumption may be incorrect.